research/object_detection/inputs.py

```diff
@@ -1041,8 +1041,11 @@ def get_single_frame(tensor_dict):
           num_frames = tf.cast(
               tf.shape(tensor_dict[fields.InputDataFields.source_id])[0],
               dtype=tf.int32)
```

Comment:
if you saw any difference in performance with fixed frame index, please indicate that in the PR description.